packages/avatar-group/test/avatar-group.test.js

```diff
@@ -90,11 +98,9 @@ describe('avatar-group', () => {
       expect(overlay.textContent).to.equal([items[2].name, items[3].abbr, 'anonymous'].join('\n'));
     });
 
-    it('should show overlay on overflow avatar click', () => {
```

Comment:
This test name was apparently a copy-paste, so I changed it to actually match the assertion.